GTG/gtk/ui/preferences.ui

```diff
@@ -0,0 +1,497 @@
+                            <property name="can_focus">False</property>
+                            <property name="label" translatable="yes">:</property>
+                            <attributes>
+                              <attribute name="weight" value="bold"/>
+                            </attributes>
+                          </object>
+                          <packing>
+                            <property name="expand">False</property>
+                            <property name="fill">True</property>
+                            <property name="padding">3</property>
+                            <property name="position">1</property>
+                          </packing>
+                        </child>
+                        <child>
+                          <object class="GtkEntry" id="minute_entry">
+                            <property name="visible">True</property>
+                            <property name="can_focus">True</property>
+                            <property name="width_chars">2</property>
+                            <property name="max-width-chars">2</property>
+                            <property name="max_length">2</property>
+                            <signal name="changed" handler="on_valid_check" swapped="no"/>
+                          </object>
+                          <packing>
+                            <property name="expand">False</property>
+                            <property name="fill">False</property>
+                            <property name="padding">3</property>
+                            <property name="position">2</property>
+                          </packing>
+                        </child>
+                        <child>
+                          <object class="GtkBox">
+                            <property name="orientation">vertical</property>
+                            <property name="valign">center</property>
+                            <property name="vexpand">false</property>
+                            <property name="visible">true</property>
+                            <property name="margin-start">10</property>
+                            <child>
+                              <object class="GtkLabel" id="time_title">
+                                <property name="can_focus">False</property>
+                                <property name="xalign">0.0</property>
+                                <property name="label" translatable="yes">Set day-start time</property>
+                                <attributes>
+                                  <attribute name="weight" value="bold"/>
+                                </attributes>
+                              </object>
+                            </child>
+                            <child>
+                              <object class="GtkLabel" id="time_label">
+                                <property name="label">Set time which will be used as a start of the new day in GTG</property>
```

Comment:
Suggestion: `Automatically refreshes the task list and adjusts WorkView`

(This one might need more polishing)